### PR TITLE
OpenSCAD: Add IncreaseToleranceFeature icon to toolbar

### PR DESCRIPTION
**Pull request by luzpaz**
Adds icon to toolbar, which lives in `src/Mod/OpenSCAD/Resources/icons/OpenSCAD_IncreaseToleranceFeature.svg`

### CODE REVIEW

**Comment by luzpaz** [comment]
@KeithSloan does this looks correct ?

File: src/Mod/OpenSCAD/InitGui.py [hunk 1]
```diff
@@ -47,11 +47,11 @@ def QT_TRANSLATE_NOOP(scope, text):
...
         commands=['OpenSCAD_ReplaceObject','OpenSCAD_RemoveSubtree',\
             'OpenSCAD_RefineShapeFeature',\
-            'OpenSCAD_IncreaseToleranceFeature', 'OpenSCAD_Edgestofaces', \
+            'OpenSCAD_IncreaseToleranceFeature', 'OpenSCAD_Edgestofaces',\
             'OpenSCAD_ExpandPlacements','OpenSCAD_ExplodeGroup']
         toolbarcommands=['OpenSCAD_ReplaceObject','OpenSCAD_RemoveSubtree',\
-            'OpenSCAD_ExplodeGroup','OpenSCAD_RefineShapeFeature']
-            #'OpenSCAD_IncreaseToleranceFeature' #icon still missing
+            'OpenSCAD_ExplodeGroup','OpenSCAD_RefineShapeFeature',\
+            'OpenSCAD_IncreaseToleranceFeature']
```
Comment (vocx-fc) [inline src/Mod/OpenSCAD/InitGui.py]: You can remove the backslash in all these. When the arguments are given inside parentheses or brackets, they don't need the backslash for line continuation. And they should preferably be visually aligned to the first element. This is,
```
toolbarcommands = ['OpenSCAD_ReplaceObject', 'OpenSCAD_RemoveSubtree',
                   'OpenSCAD_ExplodeGroup','OpenSCAD_RefineShapeFeature',
                   'OpenSCAD_IncreaseToleranceFeature']
```

Comment (luzpaz) [reply src/Mod/OpenSCAD/InitGui.py]: Done in 8bf68bd  
Thanks!

**Comment by KeithSloan** [comment]
Quick scan of code says Probably. Ideally I would want to test. I do fixes locally then make a PR request. Not sure how one checks out other people PR request.

**Comment by luzpaz** [comment]
>  Not sure how one checks out other peoples PR request.

@KeithSloan I use GUI called gitkraken (note: it's proprietary but free to use for FOSS). I can directly checkout PRs through it. I can also add others as 'remotes' (which coincidentally is featured on the wiki (in which you make a cameo! :smile:) in one of the tutorials I made a while back in the day, on how to do such a thing in gitkraken [link](https://www.freecadweb.org/wiki/Developing_FreeCAD_with_GitKraken#Following_other_FreeCAD_repositories)

You can also use `hub` (https://hub.github.com/) which is a CLI for github (note: it uses SSH so you'll to add SSH credentials to your github account) and run: 
`hub checkout https://github.com/FreeCAD/FreeCAD/pull/2616`
This will download this PR to your local and you can compile it and test. If you want to make a change, to integrate it you'll need to add me as a remote and then make a PR to my repo.  (I've not done this before with `hub`, you can read up on that separately), try the [tip from the wiki](https://www.freecadweb.org/wiki/Source_code_management#Adding_Remotes_.28AKA_other_repositories.29):
```bash
git remote add luzpaz https://github.com/luzpaz/FreeCAD
git fetch luzpaz
git checkout -b OpenSCAD_icon luzpaz/FreeCAD
```

**Comment by KeithSloan** [comment]
Well I ended up cloning your repository and checking out the OpenSCAD_icon branch
When I try to use it I get 

No module named 'OpenSCAD_rc'
Traceback (most recent call last):
  File "<string>", line 47, in Initialize

Might be because I have not built from source and just did a symbolically link to it from freecad-daily

**Comment by luzpaz** [comment]
@KeithSloan In what file is the error from ?

**Comment by KeithSloan** [comment]
Okay I built the Qt rc file as a compile from source would i.e.
pyrcc5 -o ../OpenSCAD_rc.py OpenSCAD.qrc

And it all seems to work okay.

**Comment by yorikvanhavre** [comment]
Okay then, thanks for testing!

**Comment by looooo** [comment]
```
During initialization the error "invalid syntax (<string>, line 41)" occurred in /home/lo/miniconda3/conda-bld/debug_1570459591975/_h_env/Mod/OpenSCAD/InitGui.py
Please look into the log file for further information
```

I guess some `\` are missing at the end of the continued lines here:
https://github.com/FreeCAD/FreeCAD/blob/2d5ecbd28f6617e75bffa452609e43dbb9cd2868/src/Mod/OpenSCAD/InitGui.py#L41L42

**Comment by KeithSloan** [comment]
TooTip has changed since version of luzpaz repository that I tested.

**Comment by luzpaz** [comment]
sorry about that, was a dumb mistake.

**Pull request merged**